app/views/admins/company_values/new.html.erb

```diff
@@ -0,0 +1,21 @@
+<h1>Create a new Company value</h1>
+
+<% if @company_value.errors.any?%>
+    <h2>The following errors prevented the Company value from being saved</h2>
+    <ul>
+        <% @company_value.errors.full_messages.each do |msg| %>
+            <li><%= msg %></li>
```

Comment:
It is better to not use abbreviations in code as `msg` can mean for you `message`, but for me `massage` or anything. in this case, it is pretty obvious, but in bigger methods, abbreviations can introduce mistakes, and problems with understanding what should happen. Remember that we read code ten times more than write.